.github/CONTRIBUTING.md

```diff
@@ -1,34 +1,85 @@
 # Contributing to Kitura
 
-We welcome contributions, and request you follow these guidelines.
+We welcome contributions to the Kitura project, and request you follow these guidelines.
 
- - [Raising issues](#raising-issues)
+ - [Reporting issues](#reporting-issues)
+ - [Pull Requests](#pull-requests)
+ - [Coding Standards](#coding-style)
  - [Contributor License Agreement](#contributor-license-agreement)
- - [Coding Standards](#coding-standards)
 
+---
 
-## Raising issues
+## Getting Started
 
-Please raise any bug reports on the issue tracker. Be sure to
-search the list to see if your issue has already been raised.
+To get started on contributing to Kitura, you will need to:
 
-A good bug report is one that make it easy for us to understand what you were
-trying to do and what went wrong. Provide as much context as possible so we can try to recreate the issue.
+1. Fork this repository to your account and then pull down to your machine.
 
-### Contributor License Agreement
+   `$ git clone https://github.com/YOUR_GITHUB_ID/Kitura`
 
-In order for us to accept pull-requests, the contributor must first complete
-a Contributor License Agreement (CLA). Please see our [CLA repo](http://github.com/IBM-Swift/CLA) for more information.
 
-This clarifies the intellectual property license granted with any contribution. It is for your protection as a
-Contributor as well as the protection of IBM and its customers; it does not
-change your rights to use your own Contributions for any other purpose.
+2. Make a change, usually by tackling an issue. 
 
-### Coding standards
+**Tip:** If you are on macOS, Xcode can be a great way to work on Kitura. You'll need to open a Terminal window, and then navigate to the root of the Kitura git project. Then, type the following command into Terminal:
 
-Please ensure you follow [the Kitura coding standards](https://github.com/IBM-Swift/Kitura/blob/master/Documentation/CodeConventions.md)
+   `$ swift package generate-xcodeproj`
 
-Please note:
+This will generate an Xcode Project, simply open the .xcodeproj file with:
 
- - all files must have the Apache license in the header.
- - all PRs must have passing builds for all operating systems.
+   `$ open *.xcodeproj`
+
+Unfortunately, Xcode is macOS only, and Linux will need to use a different text editor or IDE.
+
+3. Ensure all tests pass with your changes by running `swift test`. If there is any new funtionality introduced by your change, new test case(s) may be required. If you need any help writing test cases, contact us on slack.
+
+4. If they all pass, open a Pull Request following the guidelines below.
+
+## Pull Requests
+
+Issues that would be a good place to start for someone new to GitHub will have the label "Good First Issue". If there aren't any tagged issues, post in Slack and the team would be happy to help you get started.
```

Comment:
This needs to be featured prominently in "Getting Started".  Please move this to the top of that section, and link "good first issue" (lowercase) to `https://github.com/IBM-Swift/Kitura/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22`.  Supplement this with copy about how these issues are simple (docs, testing, logging, refactoring, renaming, simple bugs etc.) and will help first time contributors gain familiarity with Kitura.